src/main/java/monique/exception/DeleteException.java

```diff
@@ -0,0 +1,26 @@
+package monique.exception;
+
+
+/**
+ * The <code>DeleteException</code> class represents an exception that is thrown
+ * when an error related to deleting an item occurs. It extends from <code>MoniqueException</code>.
+ */
+public class DeleteException extends MoniqueException {
+    /**
+     * Constructs a new <code>DeleteException</code> with a default detail message.
+     * The message indicates that the item number is not allowed.
+     */
+    public DeleteException() {
```

Comment:
Would it be better to initialise DeleteExceptions with custom error messages, in case future improvements lead to more than 1 reason for a DeleteException to be thrown?